spinta/types/model.py

```diff
@@ -180,14 +180,28 @@ def link(context: Context, base: Base):
     ]
 
 
-def _parse_dtype_string(dtype: str) -> Tuple[str, List[str]]:
+def _parse_dtype_string(dtype: str) -> dict:
+    args = []
+    required = unique = False
+
+    if 'required' in dtype:
+        required = True
+        dtype = dtype.replace('required', '').strip()
+    if 'unique' in dtype:
+        unique = True
+        dtype = dtype.replace('unique', '').strip()
```

Comment:
That is not a reliable approach to parse dtype. Could you add a test for `type(required) unique` and see what happens.

First we need to handle parentheses if they are present and what goes after `)`, handle separately by splitting by space.

Also you need raise an error, if we get something unrecognized after parenthesis.

And please add tests for this function alone with various inputs.